examples/basic/areas/block.py

```diff
@@ -1,3 +1,13 @@
+"""
+Creates a block plot using Bokeh and displays it using the show() function.
```

Comment:
nit: style consistency with existing docstrings
```suggestion
""" Creates a block plot using Bokeh and displays it using the show() function.
```